_topics/en/eclipse-attacks.md

```diff
@@ -91,6 +91,9 @@ optech_mentions:
   - title: Eclair #1545 adds multiple header sources to make eclipse attacks harder
     url: /en/newsletters/2020/11/11/#eclair-1545
 
+  - title: "Review Club chat about PR #19858 to make eclipse attacks more difficult"
```

Comment:
```suggestion
  - title: "Review club discussion about PR #19858 to make eclipse attacks more difficult"
```

Reply:
That was my initial description, but it exceeds the allowed length of 75 characters.